.github/workflows/cypress_workflow.yml

```diff
@@ -28,7 +28,7 @@ on:
 
 env:
   TEST_REPO: ${{ inputs.test_repo != '' && inputs.test_repo || 'opensearch-project/opensearch-dashboards-functional-test' }}
-  TEST_BRANCH: "${{ inputs.test_branch != '' && inputs.test_branch || github.base_ref }}"
+  TEST_BRANCH: "${{ 'feature/new-home' || inputs.test_branch != '' && inputs.test_branch || github.base_ref }}"
```

Comment:
this is input that can be passed github so i believe we can just remove this. or if we create a feature branch within this repo and call it `feature/new-home` it will pull automatically.

Reply:
Spoke to Rocky offline:

This is a trick to make sure merging this PR doesn't impact other PRs while the FTR PR is open and vice versa. Creating a feature branch on FTR and pointing this PR to that, and later PRing the feature branch to FTR:
1. While this PR is being discussed and updated, the FTR tests are updated too in the feature branch; this PR will pass its tests and so will other PRs that are not pointing to this feature branch.
2. When this PR is merged, OSD `main` will continue to look at the feature branch of FTR; tests will pass because all new PRs will be pointing to the feature branch.
3. When the feature branch of FTR is PRed and merged, the new tests will be in FTR `main`; OSD tests will continue to point to the feature branch and will continue to pass.
4. A PR on OSD will reset the default FTR test branch used by the workflow to `main`; OSD tests will continue to pass since FTR `main` has the correct tests that were merged in from the feature branch.

This choreography is to make sure we never accept failing tests as a side effect of having new tests added or new functionality added.